README.md

```diff
@@ -25,14 +25,6 @@ OPTIONAL SECTIONS:
 After you're finished please remove all the comments and instructions!
 -->
 
```

Comment:
![Screenshot from 2023-02-16 01-10-49](https://user-images.githubusercontent.com/83042742/219212536-90c1dc00-f92e-4f5e-a2df-8b14ef609b6c.png)
- [ ] Kindly ensure that the The main title (e.g., "I'm Marianne ...") is a `h1` tag and a `h4` for  best practices and semantics reasons 

- [ ] Kindly make sure your Hero text design sticks as much as as possible as required for the project. Right now, the both designs have have different font-weight.

| Template Design | Your Design |
| ---------------------- | ---------------- |
| ![Screenshot from 2023-02-16 01-22-59](https://user-images.githubusercontent.com/83042742/219215443-f478456c-e877-43e8-87ae-79ad33b2f1cf.png) | ![Screenshot from 2023-02-16 01-26-21](https://user-images.githubusercontent.com/83042742/219216076-99c4d37f-7356-44c6-8f6f-0dafcaddaad0.png) |

Reply:
done